Changes.md

```diff
@@ -16,6 +16,8 @@ Fixes
 - UI :
 	- Fixed one pixel alignment issue with highlighted and disabled icons.
 	- Improved appearance of adjoined widgets.
+	- Clicking on a button no longer assigns focus to it.
```

Comment:
Please don't mix in changes from completely unrelated commits. This is explained in the very first bullet point [here](https://github.com/GafferHQ/gaffer/wiki/Editing-Changes.md).